discovery_engine/lib/src/domain/models/document.dart

```diff
@@ -69,6 +69,8 @@ class Document {
     this.isSearched = false,
   }) : timestamp = DateTime.now().toUtc();
 
+  //FIXME move this conversion to `api/models/document.dart` e.g. as an
```

Comment:
❓ i wouldn't mind if this was done already in this pr, but we can also leave it for another pr. in the latter case, please add a ticket to the backlog.

Reply:
Ok did it.